docs/data/charts/overview/overview.md

```diff
@@ -25,19 +25,23 @@ To modify the styling of charts you can rely on all the MUI styling tools, such
 
 ## Getting started
 
+:::warning
+Note we're using the `next` tag to download the latest v7 **alpha** version.
```

Comment:
I followed the previous year's warning text, could we improve it a bit? 
Reviewdog is also not happy about the use of `we` 😄

Reply:
🤔 
```suggestion
The `next` tag is used to download the latest v7 **pre-release** version.
```